Networking/Networking/Model/OrderRefund.swift

```diff
@@ -0,0 +1,112 @@
+import Foundation
+
+
+/// Represents a Order's refund Entity.
+///
+public struct OrderRefund: Decodable {
+    public let siteID: Int
+    public let refundID: Int
+    public let dateCreated: Date
+    public let amount: String
+    public let reason: String
+    public let refunded_by: Int
```

Comment:
I would suggest naming this property as `refundedBy` instead of `refunded_by`, to follow the convention used in other properties

Reply:
Fixed